src/main/java/org/gridsuite/report/server/ReportService.java

```diff
@@ -147,15 +232,26 @@ private ReportValueEmbeddable toValueEmbeddable(Map.Entry<String, TypedValue> en
         return new ReportValueEmbeddable(entryValue.getKey(), entryValue.getValue().getValue(), entryValue.getValue().getType());
     }
 
+    private String maxSeverity(ReporterModel reporter) {
```

Comment:
```suggestion
    private static String maxSeverity(ReporterModel reporter) {
```
can be static method

Reply:
Thanks, updated.